<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="background">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/json.png&quot; width=&quot;812&quot; height=&quot;824&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu::item{
    height:50px;
}</string>
    </property>
    <property name="title">
     <string>  文件(F)  </string>
    </property>
    <addaction name="button_new"/>
    <addaction name="separator"/>
    <addaction name="button_open"/>
    <addaction name="separator"/>
    <addaction name="button_save"/>
    <addaction name="button_saveas"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_network"/>
    <addaction name="save_server"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu::item{
    height:50px;
}</string>
    </property>
    <property name="title">
     <string>  编辑(E)  </string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_F6"/>
    <addaction name="action_F5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="button_new">
   <property name="text">
    <string>  新建                        Ctrl+N</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="button_open">
   <property name="text">
    <string>  打开                        Ctrl+O</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="button_save">
   <property name="text">
    <string>  保存                        Ctrl+S</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="button_saveas">
   <property name="text">
    <string>  另存为           Ctrl+Shift+N</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="button_in">
   <property name="text">
    <string>  导入</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionword_xml">
   <property name="text">
    <string>xml(.xml)</string>
   </property>
  </action>
  <action name="action_F6">
   <property name="text">
    <string>  刷新左侧视图             F6</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_F5">
   <property name="text">
    <string>  刷新右侧编辑器          F5</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Ctrl_V">
   <property name="text">
    <string>  粘贴                    Ctrl+V</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Ctrl_Z">
   <property name="text">
    <string>  撤销                   Ctrl+Z</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>  网站                          </string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>  鸣谢                          </string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionQuick_Start">
   <property name="text">
    <string>  Quick Start                   </string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_network">
   <property name="text">
    <string>  从服务器导入</string>
   </property>
  </action>
  <action name="save_server">
   <property name="text">
    <string>  储存到服务器端</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
